packages/fxa-auth-server/lib/sentry.js

```diff
@@ -161,6 +161,8 @@ async function configureSentry(server, config, processName = 'key_server') {
         new Sentry.Integrations.LinkedErrors({ key: 'jse_cause' }),
         new ExtraErrorData({ depth: 5 }),
       ],
+      // https://docs.sentry.io/platforms/node/configuration/options/#max-value-length
```

Comment:
Funny that their doc on the max value length does not mention a max.  I guess 500 is fine then?

Reply:
True. I only know 250 is the default. I imagine 500 which is the same order of magnitude should be all right... 🤔